packages/arb-validator-core/utils/rpc.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"net/http"
+
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+)
+
+func LaunchRPC(handler http.Handler, port string) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function LaunchRPC should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)